bootstrap/pkg/client/gcp/gcp.go

```diff
@@ -719,7 +723,7 @@ func (gcp *Gcp) writeGcpSecret(client *clientset.Clientset, secretName string, d
 	return err
 }
 
-func (gcp *Gcp) createGcpSecret(ctx context.Context, client *clientset.Clientset,
+func (gcp *Gcp) createGcpServiceAcctSecret(ctx context.Context, client *clientset.Clientset,
```

Comment:
please add comment for func (and below).
Like what's created

Reply:
added.

Reply:
write to GCP --> insert to cluster?

Reply:
done.